airflow/sentry.py

```diff
@@ -151,15 +151,15 @@ def enrich_errors(self, func):
             """Wrap TaskInstance._run_raw_task to support task specific tags and breadcrumbs."""
 
             @wraps(func)
-            def wrapper(task_instance, *args, session=None, **kwargs):
+            def wrapper(task_instance, *args, **kwargs):
                 # Wrapping the _run_raw_task function with push_scope to contain
                 # tags and breadcrumbs to a specific Task Instance
                 with sentry_sdk.push_scope():
                     try:
-                        return func(task_instance, *args, session=session, **kwargs)
+                        return func(task_instance, *args, **kwargs)
                     except Exception as e:
                         self.add_tagging(task_instance)
-                        self.add_breadcrumbs(task_instance, session=session)
+                        self.add_breadcrumbs(task_instance)
```

Comment:
This will make the add_breadcrumbs use a different session object --  that's not great.

I wonder if instead the place where session is passed as a positional argument should be converted to use kwargs instead?

Reply:
> I wonder if instead the place where session is passed as a positional argument should be converted to use kwargs instead?

This only solves the current problem, maybe i should parse session from positional/kwargs argument?

Reply:
@Junnplus Probably worth doing, yeah.

We already have https://github.com/apache/airflow/blob/49aa9aa2cd6e36ec211382d30a58e279755b0962/airflow/utils/session.py -- so perhaps that could be extracted somehow to not have duplicated logic?